cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithCordapp.gradle

```diff
@@ -20,8 +20,8 @@ task deployNodes(type: net.corda.plugins.Cordform) {
         name 'O=Notary Service,L=Zurich,C=CH'
         notary = [validating : true]
         p2pPort 10002
-        rpcPort 10003
         rpcSettings {
+            port 10003
```

Comment:
Has this option to specify the RPC port changed to from `rpcPort` to `rpcSettings.port` or are both valid? Wondering if we may need to update the documentation

Reply:
The plugin has been screaming at people for ages that the old `rpcPort` is deprecated and scheduled to be removed in Corda 5:
```kotlin
@Deprecated("Use {@link CordformNode#rpcSettings(RpcSettings)} instead. Will be removed by Corda V5.0.")
```
People are _supposed_ to have migrated by now :hourglass_flowing_sand:. Although to be fair, I'm not even sure where this documentation you mention actually _is_... :eyes:.

Reply:
Actually the documentation is fine seems up to date: https://docs.corda.net/docs/corda-os/4.7/generating-a-node.html